htdocs/js/instrument_form_control.js

```diff
@@ -1,26 +1,26 @@
 function notAnswered() {
-  "use strict";
-  var name = $(this).attr('name');
-  var index = this.selectedIndex;
+  'use strict';
+  let name = $(this).attr('name');
+  let index = this.selectedIndex;
   name = name.replace('_status', '');
   if (name.indexOf('_date') > -1) {
     if (index === 0) {
       $('.' + name).prop('disabled', false);
       $(this).parent().removeClass('has-warning');
-      $("#" + name).remove();
+      $('#' + name).remove();
     } else {
       $('.' + name).prop('disabled', true);
       $(this).parent().addClass('has-warning');
-      $(this).after("<div class=\"col-xs-12 warning\" id=\"" + name + "\">Any entered data will not be saved</div>");
+      $(this).after('<div class="col-xs-12 warning" id="' + name + '">Any entered data will not be saved</div>');
     }
   } else if (index === 0) {
     $('[name=' + name + ']').prop('disabled', false);
     $(this).parent().removeClass('has-warning');
-    $("#" + name).remove();
+    $('#' + name).remove();
   } else {
     $('[name=' + name + ']').prop('disabled', true);
     $(this).parent().addClass('has-warning');
-    $(this).after("<div class=\"col-xs-12 warning\" id=\"" + name + "\">Any entered data will not be saved</div>");
+    $(this).after('<div class="col-xs-12 warning" id="' + name + '">Any entered data will not be saved</div>');
```

Comment:
Is there a reason why double quotes are used within the html here and then single quotes (escaped) on other lines (for ex. line 38). I think we should just replace all double quotes and have escaped single quotes to be consistent. That shouldn't cause any problems, right?

Reply:
I actually didn't make that change because I used the new command: 
eslint --fix --ext .js, htdocs jsx modules
In javascript land it's usually done the way you're seeing it now after the eslint command.